scripts/tests/service_tests.py

```diff
@@ -697,8 +697,6 @@ def test_apm_server_acm(self):
         apm_server = ApmServer(version="7.3").render()["apm-server"]
         self.assertTrue("apm-server.kibana.enabled=true" in apm_server["command"],
                         "APM Server Kibana enabled by default")
-        self.assertTrue("apm-server.kibana.host=kibana:5601" in apm_server["command"],
```

Comment:
it is tested in another test, also render does not grab default values do it fails.